proto/testing/BUILD.bazel

```diff
@@ -102,7 +102,6 @@ go_test(
     name = "go_mainnet_test",
     size = "small",
     srcs = [
-        "ssz_static_mainnet_test.go",
```

Comment:
ssz_static tests for mainnet was deprecated. (minimal is still there). See: https://github.com/ethereum/eth2.0-specs/pull/1957

Reply:
Can you elaborate? I still see "mainnet" for phase0.yaml in that PR

Reply:
Asked upstream here: https://github.com/ethereum/eth2.0-specs/pull/1957#issuecomment-695330482

Reply:
thanks, marking this as blocked until this is resovled